frontend/packages/sdk/src/create.ts

```diff
@@ -101,34 +102,50 @@ export interface ClusterInfo {
   gasPrice: BN;
 }
 
+export interface InkQueryOk extends IEnum {
+  asInkMessageReturn: Vec<u8>
+}
+
+export interface InkQueryError extends IEnum {
+  BadOrigin: null
+  RuntimeError: string
+}
+
+export interface InkResponse extends IMap {
+  nonce: Vec<u8>;
+  result: Result<InkQueryOk, InkQueryError>;
```

Comment:
With ink! 4.0, not sure if the return result is still `Result<T, DispatchError>` or `Result<Result<T, DispatchError>, InkLangError>`

Reply:
> With ink! 4.0, not sure if the return result is still `Result<T, DispatchError>` or `Result<Result<T, DispatchError>, InkLangError>`

Should be `Result<Result<T, DispatchError>, InkLangError>` but not going to handle that in legacy codes.

Reply:
Cool. Looking forward to retire the legacy code.